go.mod

```diff
@@ -4,7 +4,7 @@ go 1.23.6
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
-	github.com/ava-labs/avalanchego v1.12.3-0.20250314173032-a3d150e0aad0
+	github.com/ava-labs/avalanchego v1.13.0-fuji-rc.2.0.20250320111141-fef07a6db96b
```

Comment:
is this the right version to depend on?

Reply:
This is avalanchego from my libevm branch (based on v1.13.0-fuji tag), notably using ava-labs/libevm instead of ava-labs/coreth/interfaces, which is accidentally relevant in this PR.
However, good comment 💯 , this made me realise, we should not bump the dependency.
On top of this, subnet-evm's libevm branch already depends on that version, although its master branch go.mod avalanchego version is pretty different than coreth's one ([diffs](https://github.com/ava-labs/avalanchego/compare/v1.12.3-name-fortuna.0...v1.12.3-rc.1))

In conclusion, I will rebase that avalanchego libevm branch such that:

- coreth-specific avalanchego libevm branch is based off `v1.12.3-name-fortuna.0`
- subnet-evm -specific avalanchego libevm branch is based off `v1.12.3-rc.1`

Reply:
This PR no longer modifies the go.mod dependencies